### ISSUE TITLE
Better docs.

### ISSUE THREAD

**Issue by BookOwl** [Tymewalk/Slither#67]
I think that the last thing that we should get done before v0.3.0 (besides the keyboard and mouse functions) is to create better docs and a tutorial, which we can host on readthedocs.

**Comment by BookOwl** [comment]
Maybe we should close #40 in favor of this.

**Comment by Tymewalk** [comment]
@BookOwl Closed #40 as it refers to the stuff we already did.

Hosting on readthedocs would be nice, and we could direct users there through PyPi, our README, and the Scratch topic.

The question now is what to do with the current help, and what kinds of examples we'll use.

**Comment by CosmicWebServices** [comment]
I could help... if u need it

**Comment by skistaddy** [comment]
`I can do the documentation`

**Comment by ghost** [comment]
`Me too please!`

**Comment by ghost** [comment]
I can turn the current help into a basic quickstart guide.

**Comment by Tymewalk** [comment]
@Omegabyte You're already on the documentation team. @bjskistad, feel free to help, once you make a PR I'll add you.

**Comment by skistaddy** [comment]
K!

**Comment by BookOwl** [comment]
We've decided to use mkdocs (#71)

**Comment by BookOwl** [comment]
I'll start working on a reference

**Comment by BookOwl** [comment]
The first-draft docs are live! http://pyslither.github.io/

**Comment by skistaddy** [comment]
Close?

**Comment by Tymewalk** [comment]
@bjskistad No. We do have a new doc location, but the docs are still low-quality.

**Comment by BookOwl** [comment]
We still need to finish the reference and add a tutorial

**Comment by Tymewalk** [comment]
Bumping as we really need docs hosted somewhere like ReadTheDocs.

Having stuff in the package is OK, but it's a little tricky to get to.

**Comment by Tymewalk** [comment]
I notice we have a nice [site](pyslither.github.io) already. Perhaps we should make that more prominent?
